source/new-ia/by-role/system-admin/clustering/studio-clustering-two-nodes.rst

```diff
@@ -318,16 +316,41 @@ nodes and verifying that your cluster size is 2:
 
    |
 
-#. Inside the MySQL client, run ``show status like 'wsrep_cluster_size'``:
+#. Inside the MySQL client, run the following:
```

Comment:
For mariaDB replication, we are now using GTID. So it might be useful to also include here these queries:

In the primary:

MariaDB [(none)]> SELECT @@GLOBAL.gtid_current_pos;
+---------------------------+
| @@GLOBAL.gtid_current_pos |
+---------------------------+
| 0-167772164-2132          |
+---------------------------+
1 row in set (0.000 sec)



In the replicas:

MariaDB [(none)]> SELECT @@GLOBAL.gtid_slave_pos;
+-------------------------+
| @@GLOBAL.gtid_slave_pos |
+-------------------------+
| 0-167772164-2145        |
+-------------------------+
1 row in set (0.000 sec)

Reply:
Also, note that the output of `show slave status` will include that info:
                    Using_Gtid: Slave_Pos
                   Gtid_IO_Pos: 0-167772164-2191